### PR TITLE
Camera event cleanup

### PR DESCRIPTION
**Pull request by favreau**

### CODE REVIEW

File: brayns/common/types.h [hunk 1]
```diff
@@ -298,13 +298,13 @@ enum MorphologyLayoutType
     ML_GRID,
 };
 
-enum CameraType
+enum class CameraType
```
Comment (tribal-tec) [inline brayns/common/types.h]: just call it 'Camera'?

Comment (favreau) [inline brayns/common/types.h]: Not in this case, it has to be consistent with the rest of the application code.

File: plugins/engines/ospray/render/OSPRayCamera.cpp [hunk 2]
```diff
@@ -32,13 +32,13 @@ OSPRayCamera::OSPRayCamera( const CameraType cameraType )
     std::string cameraAsString;
```
Comment (tribal-tec) [inline plugins/engines/ospray/render/OSPRayCamera.cpp]: Forget it, thought it was the FBS...

File: brayns/common/types.h (hunk 1 above)
Comment (tribal-tec) [inline brayns/common/types.h]: Forget it, thought it was the FBS...

File: plugins/engines/ospray/render/OSPRayCamera.cpp (hunk 2 above)
Comment (favreau) [inline plugins/engines/ospray/render/OSPRayCamera.cpp]: Ok. thx.

**Pull request merged**